test/cppsuite/tests/run.cpp

```diff
@@ -108,6 +109,7 @@ print_help()
     std::cout << std::endl;
     std::cout << "OPTIONS" << std::endl;
     std::cout << "\t-h Output a usage message and exit." << std::endl;
+    std::cout << "\t-H Specifies the database home directory." << std::endl;
```

Comment:
Same, A->Z, btw, the `-h` should probably be before the `-f` option.

Reply:
[eb6d460](https://github.com/wiredtiger/wiredtiger/pull/8484/commits/eb6d460cf9f91e632783d4cbe5537064096d2264)